src/Codacy/Coverage/Command/Clover.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Codacy\Coverage\Command;
+
+use Symfony\Component\Console\Command\Command as ConsoleCommand;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use Codacy\Coverage\Parser\CloverParser;
+use Codacy\Coverage\Util\JsonProducer;
+use Codacy\Coverage\Util\GitClient;
+use Codacy\Coverage\Util\CodacyApiClient;
+
+/**
+ * Class Clover
+ *
+ */
+class Clover extends ConsoleCommand {
+
+    protected function configure()
+    {
+        $this
+            ->setName("clover")
+            ->setDescription("Send coverage results from clover.xml format")
+            ->addArgument(
+                "path_to_coverage_results",
+                InputArgument::OPTIONAL,
+                "Path where coverage results saved"
+            )
+            ->addOption(
```

Comment:
I am trying to make sense of this. Why should anyone want to specify the commit?